### ISSUE TITLE
support app?

### ISSUE THREAD

**Issue by forraiv** [RassK/homey-komfovent#4]
Is this project dead? 
Can anyone recommend an alternative if the development of this application is discontinued?

**Comment by RassK** [comment]
@forraiv yes, it's dead. It was mostly doing only data scrubbing. To actually control your Komfovent AHU a modbus based integration is necessary. 

I have moved on from Homey to Fibaro HC3 and then to HomeAssistant. So far I can say that HomeAssistant has the best modbus support and it works very well with Komfovent.
